### ISSUE TITLE
Mock data generation with multiple columns 

### ISSUE THREAD

**Issue by bzkrouse** [GSK-Biostatistics/tfrmt#134]
```
 tfrmt(
   group = group,
   label = label,
   param = param, 
   column = c(col1, col2),
   body_plan = body_plan(
     frmt_structure(group_val = ".default", label_val = ".default", frmt("xx.x"))
   )) %>%
   print_mock_gt()
```

produces: 
![image](https://user-images.githubusercontent.com/14199771/173920861-436dfa75-ff58-4665-9ebc-0d30f9848fbd.png)

**Comment by statasaurus** [comment]
Isn't this resolved now

**Comment by thebioengineer** [comment]
The mock data code only creates a col2. I have an idea on how to fix this, and can add it into #154

**Comment by thebioengineer** [comment]
See commit db79658
